tests/test_tax_engine.py

```diff
@@ -33,12 +33,14 @@
 
 class TestTaxEngine(unittest.TestCase):
     _good_input_configuration: Configuration
+    _good_input_allow_negative_configuration: Configuration
```

Comment:
Let's use a fully spelled-out name (it will make it easier to understand it's purpose):
```suggestion
    _good_input_allow_negative_balance_configuration: Configuration
```